cirq-core/cirq/transformers/routing/mapping_manager_test.py

```diff
@@ -143,3 +143,18 @@ def test_shortest_path():
     one_to_four[1], one_to_four[2] = one_to_four[2], one_to_four[1]
     assert mm.shortest_path(q[1], q[4]) == one_to_four
     assert mm.shortest_path(q[1], q[2]) == [q[1], q[2]]
+
+
```

Comment:
Add tests for `value_equality` using `cirq.testing.EqualsTester`. Test the following different cases:
1) Different graph types as input: Undirected graphs with different ordering of edge insertions, directed graphs with edge insertions same as undirected graph (should be unequal), graphs with isolated nodes (nodes with degree 0)
2) Two different mapping managers with same initial graph and initial mapping but different current states (due to applying swaps etc.) so that `self._map` is different for both.